### ISSUE TITLE
Update to laravel 6.0

### ISSUE THREAD

**Issue by michaloravec** [nunomaduro/collision#70]
Hi I have problem to upgrade to laravel 6.0.

When I run composer update I get:
![image](https://user-images.githubusercontent.com/22096293/68927054-33d81500-0787-11ea-8cd2-5352ef15298e.png)

If I put this package to dont-discover like this:
![image](https://user-images.githubusercontent.com/22096293/68927197-887b9000-0787-11ea-9184-bfcf03600f06.png)

Every is OK. 

On Laravel 5.8 I don't have this problem.

**Comment by nunomaduro** [comment]
@michaloravec What version of collision are you using?

**Comment by michaloravec** [comment]
In composer.json I have this:
![image](https://user-images.githubusercontent.com/22096293/68928827-4fddb580-078b-11ea-9d11-023074491870.png)

**Comment by nunomaduro** [comment]
Let's debug this. Perform a composer update --verbose without the dont discover stuff.

**Comment by michaloravec** [comment]
![image](https://user-images.githubusercontent.com/22096293/68929215-32f5b200-078c-11ea-91b7-0425cfe479dc.png)
And before just reading other packages

**Comment by michaloravec** [comment]
Full output [composer.pdf](https://github.com/nunomaduro/collision/files/3850596/composer.pdf)

**Comment by nunomaduro** [comment]
Sorry I don't have a windows machine. Can't help you debug this.

Keep in mind, that collision is working correctly in fresh new laravel 6.4 applications. So I have no idea what your error could be. Sorry.

**Comment by zubairniazi** [comment]
Hi, @nunomaduro 

Is there a way to prevent warnings in the Laravel 6.* project? I installed a new Laravel 6 project and after running **composer install**, I got below output:

```
Package jakub-onderka/php-console-color is abandoned, you should avoid using it. Use php-parallel-lint/php-console-color instead.
Package jakub-onderka/php-console-highlighter is abandoned, you should avoid using it. Use php-parallel-lint/php-console-highlighter instead.
```

It seems the latest version (v4.2) got the fix but not compatible with the Laravel 6. I hope you have some solution.

**Comment by nunomaduro** [comment]
Honestly just ignore that warning. It means nothing relevant.

**Comment by zubairniazi** [comment]
Thank you for your quick response. :)
My concern is security; that's why I posted the issue and wants my composer to run sweetly. Am I safe from these in terms of security threats?

**Comment by nunomaduro** [comment]
Those are dev dependencies, so those dependencies don't end up in production. 

TL; DR: is safe.
